package.json

```diff
@@ -11,6 +11,7 @@
     "file-extension": "^4.0.0",
     "ipfs-api": "^17.1.3",
     "ipfs-geoip": "^2.3.0",
+    "ipfs-logo": "^0.2.0",
```

Comment:
You should not need this anymore. Make sure to rebase master onto this branch @VictorBjelkholm